src/Mailer/Message/CitizenInitiativeActivitySubscriptionMessage.php

```diff
@@ -67,7 +68,7 @@ public static function create(
         foreach ($recipients as $recipient) {
             $message->addRecipient(
                 $recipient->getEmailAddress(),
-                self::fixMailjetParsing($recipient->getFullName()),
+                MailjetUtils::fixMailjetParsing($recipient->getFullName()),
```

Comment:
same here.

Reply:
yes, all subjects are fixed this way. same as above, will move it in the ApiClient logic